components/Paginated/Paginated.module.scss

```diff
@@ -1,11 +1,7 @@
-@import 'colors', 'fonts', 'layout';
+@import 'colors','fonts','layout';
 
-.relative {
-  position: relative;
-}
 
-.container {
-  bottom: calc(-50vh - 10px);
+.relative {
```

Comment:
Note that `.container` disappeared, is this intentional?

Reply:
.container is not more needed in paginated.module.scss